MWStoryGitFile.txt

```diff
@@ -24926,7 +24926,7 @@ Text inside of the PIPE signs (these things -> |text| ) are varibales, please do
 "Sun Jul 11 15:57:18 GMT+0200 2021_15007"MainText"First I would like to see it!"
 "Sun Jul 11 15:57:18 GMT+0200 2021_15008"MainText"Alright, we better build up this device in the bathroom."
 "Sun Jul 11 15:57:18 GMT+0200 2021_15009"MainText"This... I think is not a good idea. I barely have any space left in there."
-"Sun Jul 11 15:57:18 GMT+0200 2021_15010"MainText"Okay... then... ahm... do you have something to put underneath it? Something water resistant?"
+"Sun Jul 11 15:57:18 GMT+0200 2021_15010"MainText"Okay... then... ahm... do you have something to put underneath it? Something waterresistant?"
```

Comment:
You reintroduced some spelling errors here which have been resolved by dubsington and the others already >_<